internal/version/version.go

```diff
@@ -19,9 +19,11 @@ import (
 	"github.com/cri-o/cri-o/utils"
 )
 
+// Version is the version of the build.
+const Version = "1.19.0"
```

Comment:
I think for now this should be `1.19.0-dev` to make clear we haven't released any 1.19  yet

Reply:
Fixed :)